packages-next/keystone/src/lib/createKeystone.ts

```diff
@@ -257,6 +257,10 @@ export function createKeystone(config: KeystoneConfig): KeystoneSystem {
         },
         schema: graphQLSchema,
       } as KeystoneGraphQLAPI<any>,
+      // Note: These two fields let us use the server-side-graphql-client library.
+      // We may want to remove them once the updated graphQL API is available.
```

Comment:
For upgraders, I'm seriously considering a config flag like `includeLegacyAPIs` that includes stuff like this, but is off by default. Sort of like a reverse "experimental" flag (which tailwind does really well)

Thinking:
* Want to hide this when there are better ways, so new users only see _the best way_ we have for them to do a thing
* Don't want to force people into upgrading to new changed APIs when all we need to do is continue passing internals through context, if what they've got will continue working

Just a not since you mention it here, not a thing for this PR

Reply:
Yep, that's a plausible pattern, let's keep that on the table as an option and try to call out where we'd want to see it 👍